### DISCUSSION TITLE
Laravel Index Suggestions

### DISCUSSION THREAD

**Discussion by colivinggit** [HassanZahirnia/laravel-package-ocean#144] [Package Suggestions]
A package which analyzes all queries, migrations etc, to give a list of suggested indexes and improvements for your database.

Will let people with low DB knowledge save a lot of time, and help the internet become faster.

**Comment by HassanZahirnia** [comment]
Hey, I'm guessing that's a package idea? rather than an actual package suggestion? Unfortunately this project doesn't cover this topic but thanks for making effort!
